apps/src/StudioApp.js

```diff
@@ -146,8 +146,8 @@ var StudioAppClass = function () {
 
   this.MIN_WORKSPACE_HEIGHT = undefined;
 };
-
-module.exports = StudioAppClass;
+var studioAppSingleton = new StudioAppClass();
+module.exports = studioAppSingleton;
```

Comment:
Couple of thoughts.
(1) Seems like it would be nice to be able to access StudioAppClass externally, even if we don't need to do so anywhere right now
(2) Should we now rename StudioAppClass -> StudioApp?
(3) If we're exporting the singleton, it feels like we should maybe rename the file to studioAppSingleton?

What do you think?